debian/control

```diff
@@ -1,21 +1,35 @@
 Source: aeneas
 Section: python
-X-Python-Version: >= 2.7
 Priority: optional
 Maintainer: Alberto Pettarin <alberto@readbeyond.it>
 Build-Depends: debhelper (>= 9.0.0),
                dh-python,
                python-all-dev,
                python-setuptools,
                python-numpy, python-lxml, python-bs4,
+               python3-all-dev,
+               python3-setuptools,
+               python3-numpy, python3-lxml, python3-bs4,
                libasound2-dev, libsndfile1-dev, libespeak-dev
-Standards-Version: 3.9.5
+Standards-Version: 4.1.4
 Homepage: https://github.com/readbeyond/aeneas
 
 Package: python-aeneas
 Architecture: any
 Depends: ${misc:Depends}, ${python:Depends}, 
```

Comment:
Needs `${shlibs:Depends}` to work around lintian error: `missing-dependency-on-libc`

Reply:
done.